src/main/java/com/redhat/pantheon/model/Product.java

```diff
@@ -0,0 +1,31 @@
+package com.redhat.pantheon.model;
+
+import java.util.Calendar;
+
+import javax.annotation.Nonnull;
+
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Default;
+
+import com.redhat.pantheon.model.api.SlingResource;
+
+public class Product extends SlingResource {
+
+    @Default(values = "pantheon/product")
```

Comment:
Just FYI: This annotation does nothing. It's just a reminder of what the default type should be.

Reply:
ack